src/Pages/Topics.jsx

```diff
@@ -18,6 +18,35 @@ const Topics = () => {
   return (
     <div>
       <h1>Topics: {params.topics}</h1>
+      {console.log(params.topics)}
+      {articles
+        .filter((article) => article.topic === params.topics)
```

Comment:
At the moment we're fetching all the articles from our api and filtering through these to only display the articles of a single topic but this has meant the Articles.jsx and Topics.jsx are nearly identical. 

You can rework your fetch request so that you can pass in a topic and it will only fetch articles from the backend with that topic but if no topic is passed in, it will fetch all articles :)